### PR TITLE
feat: [SRM-10805]: templates implementation for verify step

### PR DESCRIPTION
**Pull request by harshilbits**
##### Summary:
1. Templates implementation for verify step

##### Jira Links:
1. https://harness.atlassian.net/browse/SRM-10805

##### Screenshots:

You can use the following comments to re-trigger PR Checks

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress: `trigger cypress`
- Fix Prettier - `fix prettier`

### CODE REVIEW

**Comment by harshilbits** [comment]
retrigger titlecheck

File: src/modules/85-cv/pages/monitored-service/CVMonitoredService/MonitoredServiceInputSetsTemplate.utils.ts [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+        clonedList = getNestedFields(spec[item[0]], clonedList, `${basePath}.${item[0]}`)
+      }
+    } else {
+      clonedList.push({ name: item[0], path: `${basePath}.${item[0]}`, value: item[1] })
+    }
+  })
+  return clonedList
+}
+
+export const getLabelByName = (name: string, getString: UseStringsReturn['getString']): string => {
```
Comment (deepesh-ui) [inline src/modules/85-cv/pages/monitored-service/CVMonitoredService/MonitoredServiceInputSetsTemplate.utils.ts:49]: If we go this way we will need to hardcode fields name for all health source.
Lets discuss on this .

File: src/modules/85-cv/pages/monitored-service/CVMonitoredService/MonitoredServiceInputSetsTemplate.utils.ts [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+    case 'category':
+      return `Category for ${getString('cv.monitoringSources.riskCategoryLabel')}`
+    case 'metricType':
+      return `Metric type for ${getString('cv.monitoringSources.riskCategoryLabel')}`
+    default:
+      return name
+  }
+}
+
+export const getLabelByNameForTemplateInputs = (name: string, getString: UseStringsReturn['getString']): string => {
```
Comment (deepesh-ui) [inline src/modules/85-cv/pages/monitored-service/CVMonitoredService/MonitoredServiceInputSetsTemplate.utils.ts:74]: If we go this way we will need to hardcode fields name for all health source.
Lets discuss on this .

Comment (harshilbits) [reply src/modules/85-cv/pages/monitored-service/CVMonitoredService/MonitoredServiceInputSetsTemplate.utils.ts:74]: sure.

**Pull request merged**